components/TimeSeriesGroup.client.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <!-- Per each time series group, use one accordion component with one item, so we can easily initialise them all as open with active-item-key -->
+  <CAccordion
+    :style="accordionStyle"
+    class="time-series"
+    :active-item-key="props.open ? props.seriesGroup.id : undefined"
+  >
+    <CAccordionItem :item-key="props.seriesGroup.id" class="border-0 position-relative">
+      <div v-if="props.open" class="switch-container position-absolute">
+        <CFormSwitch
+          :id="`${props.seriesGroup.id}DailySwitch`"
+          v-model="isDaily"
+          label="New per day"
+        />
+      </div>
+      <CAccordionHeader class="border-top" @click="$emit('toggleOpen')">
+        <span aria-describedby="labelDescriptor">{{ activeSeriesMetadata?.label }}</span>
+        <span id="labelDescriptor" class="visually-hidden">{{ activeSeriesMetadata?.description }}</span>
+        <CTooltip
+          v-if="activeSeriesMetadata?.description"
+          :content="activeSeriesMetadata.description"
+          placement="top"
+        >
+          <template #toggler="{ togglerId, on }">
+            <CIconSvg class="icon smaller-icon opacity-50 ms-2">
+              <img src="~/assets/icons/circleQuestion.svg" :aria-describedby="togglerId" v-on="on">
+            </CIconSvg>
+          </template>
+        </CTooltip>
```

Comment:
I've just merged the Parameter Help PR, which provides a TooltipHelp component which can be used in place of this - it was replaced in the old time series component in that branch, but will probably need to be manually resolved here because of the time series component refactor